script/ci.sh

```diff
@@ -18,3 +18,7 @@
 
 bazel build //...
 bazel test //... --test_output=errors
+
+# Push benchmark binary image to cloudesf-testing GCR.
+gcloud config set core/project cloudesf-testing
```

Comment:
Not sure it is a good idea to list this project here at github??

Reply:
Yeah, I was also a bit skeptical about this, but we also have it in [esp-v2](https://github.com/GoogleCloudPlatform/esp-v2/blob/57771a68d62bfa18425016c2d66fb34d5da93abc/prow/gcpproxy-build.sh#L28). Maybe there is a way to store this project-id in the GitHub secrets, or maybe OSS Prow has a way to hide this. 
I will look into these options once I got the CI passed :) It's still failing to push this image.

Reply:
The Prow job runner doesn't have the permissions to `cloudesf-testing` project. I will remove this CI integration from this PR and keep it as a backlog. We can do this along with b/242090910.